.buildkite/steps/cleanup.sh

```diff
@@ -44,15 +45,15 @@ aws s3api list-buckets \
   --output text \
   --query "$(printf 'Buckets[?CreationDate<`%s`].[Name]' "$cutoff_date" )" \
   | xargs -n1 \
-  | grep -E 'buildkite-aws-stack-test-(\d+-)?managedsecrets' \
+  | grep -E 'buildkite-aws-stack-test-.*-managedsecretsbucket' \
   | xargs -n1 -t -I% aws s3 rb s3://% --force
 
 echo "--- Deleting old cloudformation stacks"
 aws cloudformation describe-stacks \
   --output text \
   --query "$(printf 'Stacks[?CreationTime<`%s`].[StackName]' "$cutoff_date" )" \
   | xargs -n1 \
-  | grep -E 'buildkite-aws-stack-test-\d+' \
+  | grep -E 'buildkite-aws-stack-test-(linux|windows)-(amd64|arm64)-\d+' \
```

Comment:
Looks like `\d` etc only ever worked on BSD grep (macOS local dev), not GNU grep on the real CI servers.

```
bash-4.2# grep --version | head -n1
grep (GNU grep) 2.20
bash-4.2# echo foo-123 | grep -E 'foo-\d+' | wc -l
0
```

```
pda@paulbookpro ~ $ grep --version | head -n1
grep (BSD grep) 2.5.1-FreeBSD
pda@paulbookpro ~ $ echo foo-123 | grep -E 'foo-\d+' | wc -l
       1
```

Reply:
Fixed in 6f94605